src/Appwrite/Utopia/Response/Model/Func.php

```diff
@@ -81,16 +81,16 @@ public function __construct()
                 'example' => '5 4 * * *',
             ])
             ->addRule('scheduleNext', [
-                'type' => self::TYPE_INTEGER,
-                'description' => 'Function next scheduled execution date in Unix timestamp.',
-                'default' => 0,
-                'example' => 1592981292,
+                'type' => self::TYPE_DATETIME,
+                'description' => 'Function next scheduled execution date in Datetime.',
```

Comment:
```suggestion
                'description' => 'Function's next scheduled execution time in Datetime.',
```

I suggest saying time instead of date because it's much more granular than days; it specifies time of day,